python/ray/serve/model_wrappers.py

```diff
@@ -248,6 +248,14 @@ async def predict(self, inp):
         """Perform inference directly without HTTP."""
         return await self.predict_impl(inp)
 
+    async def reconfigure(
+        self,
+        checkpoint: Union[Checkpoint, str],
+        predictor_cls: Union[str, Type["Predictor"]],
+    ) -> Type["Predictor"]:
+        """Returns a new Predictor from Checkpoint"""
+        return await predictor_cls.from_checkpoint(checkpoint, predictor_cls)
```

Comment:
the return result is actually not used here. I would recommend doing the following instead:
- return nothing
- set self.model to the value of from checkpoint
- drop the await, from_checkpoint is a synchronous call.

Reply:
Applied suggested changes. Please take a look when you get a chance.